### PR TITLE
Add basic Travis integration

### PR DESCRIPTION
**Pull request by stevejb71**

### CODE REVIEW

**Comment by stevejb71** [comment]
Fails...

The command "bin/fetch-configlet" exited with 0.
0.01s$ bin/configlet lint --track TRACK_ID .
Error: unknown flag: --track
Usage:
  bin/configlet lint <path/to/track> [flags]

**Comment by stevejb71** [comment]
Fails with
```
The command "bin/fetch-configlet" exited with 0.
0.01s$ bin/configlet lint --track TRACK_ID .
Error: unknown flag: --track
Usage:
  bin/configlet lint <path/to/track> [flags]
```

File: travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: reasonml
```
Comment (tejasbubane) [inline travis.yml]: I don't think travis supports reasonml - https://docs.travis-ci.com/user/languages/

**Comment by stevejb71** [comment]
I'll decline this one in favour of yours.

**Comment by tejasbubane** [comment]
Sorry @stevejb71 I did not mean to step on your toes, I had a similar implementation from one of my previous [repos](https://github.com/tejasbubane/adventofcode-2017) - so gave it a try.

**Comment by stevejb71** [comment]
No problem. I'd only just started, and it looks like you have more
experience here.

**Pull request closed without merging**